perfil/core/management/commands/__init__.py

```diff
@@ -189,13 +188,49 @@ def create_parser(self, *args, **kwargs):
 
     def add_arguments(self, parser):
         parser.add_argument("csv", help="Path to CSV file")
+        parser.add_argument(
+            "clean-previous-data",
+            default=False,
+            nargs="?",
+            help=(
+                "Creates politicians for all affiliations "
+                "regardless if the politician exists or not."
+            ),
+        )
+
+    def delete_all_objects(self):
+        all_objs = self.model.objects.all()
+        minor_pk = all_objs.aggregate(Min("pk")).get("pk__min")
+        major_pk = all_objs.aggregate(Max("pk")).get("pk__max")
+
+        if not minor_pk or not major_pk:
+            print(f"There are no data to delete in {self.model._meta.verbose_name}")
+            return
+
+        kwargs = {
+            "desc": f"Deleting data from {self.model._meta.verbose_name}...",
+            "total": major_pk - minor_pk + 1,
+            "unit": "objetcs",
+        }
+        offset = minor_pk
+        bulk_size = 100
+        with tqdm(**kwargs) as progress_bar:
+            while offset <= major_pk:
+                self.model.objects.filter(
+                    pk__gte=offset, pk__lt=offset + bulk_size
+                ).delete()
+                offset += bulk_size
+                progress_bar.update(bulk_size)
```

Comment:
Boa pergunta, @giuliocc ! Eu fiz dessa forma para evitar esse problema:  [Não é possível deletar mais de 999 objetos se há uma relação apontando para eles](https://code.djangoproject.com/ticket/16426).
Eu não sei se isso _ainda_ é um problema, mas era para evitar dor de cabeça com isso hehe :)

----
### English
Nice question, @giuliocc ! I did it this way to avoid this problem: [Cannot delete more than 999 things if there is a relation pointing to them](https://code.djangoproject.com/ticket/16426)
I don't know if it's _still_ a problem, but I did to avoid headaches with it haha :)

Reply:
### Português

Sem problema :) Era mais por curiosidade mesmo!

### English

No problem :) It was just out of curiosity!